gnovm/cmd/gno/test.go

```diff
@@ -226,10 +234,15 @@ func gnoTestPkg(
 
 	var errs error
 
+	mode := tests.ImportModeStdlibsOnly
+	if cfg.withNativeFallback {
+		// XXX: display a warn?
```

Comment:
I would say yes because this might end up trying to read a file from disk or doing other illegal actions inside the VM that will fail.

Reply:
With CI and reviewers catching any errors, this isn't high-risk. I'm still figuring out the next steps for warnings and logs. For now, let's keep the comment as is.